docs/cristin.ttl

```diff
@@ -832,6 +847,54 @@ It is a self reference.""" ;
     :isignored   false .
 
 
+:finansieringslopenr
+    a            :cristinfield ;
+    rdfs:range   xsd:integer ;
+    rdfs:domain  :FINANSIERING_VARBEID_Class ;
+    rdfs:comment """Internal Cristin ID of a funding entry / research grant.
+    Can be dereferenced at https://api.cristin.no/v2/fundings/<finansieringslopenr>
+    """ ;
+    :isignored   true .
+
+:finansieringskildekode
+    a            :cristinfield ;
+    rdfs:range   xsd:string ;
+    rdfs:domain  :FINANSIERING_VARBEID_Class ;
+    rdfs:comment """Code representing the funding body that awarded the research grant.
+    Possible values include "EU","NFR".
```

Comment:
The code for financing source is probably its own Thing, so:

:EU a :Finansieringskilde … but maybe overkill